apollo-api/src/main/java/com/apollographql/apollo/api/GraphqlUpload.java

```diff
@@ -0,0 +1,13 @@
+package com.apollographql.apollo.api;
+
+import java.io.File;
+
+public class GraphqlUpload {
```

Comment:
Should we call it `GraphQLUpload `?

Reply:
Is just `Upload` or `FileUpload` ok? any suggestion...